tests/test_wedowind.py

```diff
@@ -0,0 +1,170 @@
+"""Tests for the `WeDoWind` example wind farm."""
+
+import shutil
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from examples.helpers import setup_logger
```

Comment:
below suggestion is to import wind_up as well so you can test version as per other suggestions. Otherwise the test will fail as soon as version is not `0.1.11`
```suggestion
import wind_up
from examples.helpers import setup_logger
```